\documentclass[onecolumn,10pt]{asme2ej}


\usepackage{epsfig} %% for loading postscript figures
\usepackage{amsmath}
\usepackage{graphicx}
\usepackage{mathtools}  
\mathtoolsset{showonlyrefs}  
\usepackage{amsbsy}
\usepackage{siunitx}
\usepackage[graphicx]{realboxes}
\usepackage[unicode]{hyperref}
\usepackage{rotating}
\usepackage{amsmath}
\usepackage{tabularx}
\usepackage{epsfig} 
\usepackage{booktabs}
\usepackage{multirow}

\usepackage{adjustbox}
\usepackage{changepage}

\usepackage{geometry}
\geometry{
	a4paper,
	total={170mm,257mm},
	left=6cm,
	top=6cm,
}

\textwidth = 14 cm


\usepackage{fancyhdr}
\pagestyle{fancy}
%\fancyhead[RE,LO]{Essay 4 for module 7} % clear all header fields
\renewcommand{\headrulewidth}{0.4pt} % no line in header area
\fancyfoot{} % clear all footer fields
\fancyfoot[LE,RO]{\thepage}           % page number in "outer" position of footer line
%\renewcommand{\footrulewidth}{0.4pt}% default is 0pt
%\fancyfoot[RE,LO]{Word count: 1 483 words.} % other info in "inner" position of footer line

%\title{In what ways might the MR Linac revolutionise radiotherapy?}

%\author{Christie Palmer\\
%        Student Member of MSc Radiation Biology\\
%        Department of Oncology\\
%        University of Oxford\\
%        email: christie.palmer@oncology.ox.ac.uk}



\begin{document}
	
	\tableofcontents
	\newpage

%\maketitle    
%\par\noindent\rule{\textwidth}{0.4pt}

\textit{"{\small The development of a tumour is guided by interactions in the microenvironment including the dynamics of interstitial fluid, and microcirculation of blood vessels. This complexity leads to tumour heterogeneity and heterogeneity between tumours of various origins and at differing stages of progression. Development of new therapeutics relies on screening in an accurate recapitulation of the tumour microenvironment, which is currently not reflected in the majority of current preclinical models. Therapeutics. After in vivo studies ~60\% of cancer therapeutics proven efficient in two-dimensional in vitro models are excluded due to ineffectiveness. Previously in our lab, a prototype bioreactor culture platform capable of supporting 3D “micro-tissues” formed either from established cancer cell lines or from small amounts of patient tissue has been developed. The small amount of tissue needed makes this system ideal for sustaining cancers isolated from the brain or pancreas as clinical samples are rare. The bioreactor design incorporates the perfusion of culture media to mimic the flow of the interstitial environment of tumours and resultant fluid dynamics. Effects of therapeutics with potential impact on the tumour microenvironment will be observed on the micro-tissue, which allows for screening and validation of cancer therapeutics in a physiologically relevant microenvironment prior to in vivo testing.}"}


\section{Introduction}
\subsection{DYRK1A}
\subsubsection{Down syndrome and green tea}
Where the intuition comes from, how DS invidividuals all have mental retardation to some extent, found of the genes in the overexpressed (on Chr21) 


Extremely sensitive and dose-dependent gene: over or under expression leads to decreased neonatal viability.  

DYRK1A is an inhibitor of proliferation so treatment with inhibitors should increase proliferation rates? 
Its loss of function leads to overproliferation and mass cell death (\cite{FernAndez-Martinez}).
DYRK1A overexpression leads to Hippo inhibition (which is a break on organ expansion)! Depending on the context, completely different effect.

\textbf{ATTENTION: "overexpression of DYRK1A is necessary to induce neural differentiation" which would mean that treatment with inhibitors should block any differentiation (we should have less markers for it if cells have been treated).} 

what we know about it (neurons, etc), how DYRK1A inhibitors came about.
\subsubsection{Radiation and DYRK1A}
DYRK1A is a negative regulator of intrinsic death (= mitochondria, check caspase 9). So inhibition should SENSITISE cells to radiation (would be nice to check for apoptosis). 


apoptosis
\subsection{Culture methods}
\subsubsection{2D and 3D}
\subsubsection{Perfusion models}
DO MORE RESEARCH.

\subsection{Aims and Objectives}

1) Development and design of optimised bioreactors.

2) Effects of combination of radiation with inhibitors on traditional culture methods (2D and 3D)

3) Combination of engineering and biology (so do you get better/different results with perfusion for instance). 

- % https://ac.els-cdn.com/0026286281900844/1-s2.0-0026286281900844-main.pdf?_tid=db29d00f-216f-4977-8d4f-82edc253603f&acdnat=1525444809_4f2a85007112baf3922d2e73d1db7631
The average speed in a brain capillary (in rodant rat) is about 0.5-15mm/s

- https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2738865/
show the relevance with these neural stem cells still present in adult (but also in kid) maybe do some further research on like the number of people undergoing radiotherapy which affects the brain (find epidemiologic study that shows to what extent the risk of secondary malignancies etc. is).

- hypothesis: DYRK1A might be radiomodifier (either sensitise or protect) but depending on its expression levels, it may be neurotoxic?
- the main focus is on using different in vitro models to optimise the correlation between in vitro and in vivo results.

- to buttress this subject, DYRK1A inhibitors will be used, along with radiotherapy to - equally - show its relevance in cancer (as little is known about this protein so far). 

- do a little introduction on DYRK1A and the "pistes" we know and care to explore (apoptosis, differentiation - so inhibitors should block differentiation? While nothing should lead to further differentiation?)

- DYRK1A is upstream of the Hippo pathway which stops proliferation and promotes differentiation... 

- DYRK1A overexpression (21 chromosome) leads to early induction of differentiation (too little mature neurons - not enough time to proliferate). 

- DYRK1A blocks the cells cycle 

--> post radiation, you need Rb activation (to prevent proliferation, block the cell cycle and repair) which is in parts related to DYRK1A. This may protect cancer cells (which have increased DYRK1A expression). In this context, \textbf{DYRK1A inhibitors would sensitise the cells to radiation}. 

- treating the cells with DYRK1A inhibitor in a differentiation medium should prevent (or at least hamper) the differentiation process since overexpression of DYRK1A leads to 


- 18.04: \textbf{overexpression of DYRK1A leads to inhibition of proliferation (and induction of differentiation), but since it's a "dosage sensitive gene", this doesn't mean that inhibition of DYRK1A leads to more proliferation and less differentiation? It would only do so on overexpressing DYRK1A cells. --> je pars en vrille, but maybe overexpressing DYRK1A in certain cells could make them more robust to radiation? MAY CHANGE}

- mention that it'd be nice to rescue the DYRK1A but lol ain't got time (time constraints).


\section{Materials and methods}

\subsection{Cell lines and culture conditions}
The neural stem cell line (REN VM) cells (generous gift from Delia Koennig) were cultured in ReNcell NSC Maintenance Media (NSC MM; Merck) conjugated with growth factors (EGF and FGF) to allow their proliferation. The differentiation of these neural stem cells simply required their sustained culture in growth factor-free Maintenance Medium. 

2D culture:
Laminin (from Sigma Aldrich), plate with 20ug/mL, we add 5ml, leave on for 4hrs in incubator, then we plate the cells at 2mil/T75 flask, following these confluencoes.
 

\subsection{DYRK1A Inhibitors}

Leucettine (L41) from Biovision.

 INDY from Sigma Aldrich : benzothiazole derivative showing a potent ATP-competitive inhibitory effect with IC50 and Ki values of 0.24 and 0.18 μM, respectively.
 
 Harmine from Cayman Chemical is a $\beta$-carboline alkaloid which has the drawback of targeting very strongly for monoamine oxidase A (although this does not affect phosphorylation of Tau, key in DYRK1A activity \cite{Frost}). It is an ATP competitive inhibitor that binds to the active conformation of the kinase domain (type I inhibitor) \cite{Ruben2015}.


All inhibitors were diluted in DMSO (Sigma-Aldrich) to obtain the stock concentrations in Table X, aliquoted and kept in -20C.

\begin{table}[h]
	\centering
	\caption{My caption}
	\label{my-label}
	\begin{tabular}{ll}
		Inhibitor & Stock concentration \\
		L41       &    10 mM                 \\
		INDY      &    10 mM                 \\
		Harmine   &    5 mM             
	\end{tabular}
\end{table}

Maybe should write down the IC50 values and make a note of these in here? Maybe not here. 



\subsection{Transient siRNA knockdown of DYRK1A}
Doing electroporation, using reverse transfection (must detach the cells first and count). 
Number of cells: 250 000/well (5) = 1,250e6 cells which we centrifuge in a falcon, and resuspend. To know how much volume to resuspend in, we need 20ul of nucleofector solution per sample, so 20 * 5 = 100 uL of resuspension. 
We want 6pmol/sample (6e12mol).
Using SiLuc at 25uM. For SiRNA, we want 1 concentration: 300nM = 6pmol. We dilute the stock *2.5 to get [SiLuc] = 10uM and we take 0.6ul.
Add 0.4ug of GFP per sample (0.5/ul = 0.8ul) .  
Using SiDYRK1A stock at 20uM. For SiRNA, we want three concentrations: D100, D200 and D300nM, so we dilute the stock twice to get [SiDYRK1A] = 10uM. Then, we get 300nmol - 1L, how much in 20uL (same for 100, 200).
300 nmol = 6 pmol -> 0.6ul.
200 nmol = 4 pmol -> 0.4ul.
100 nmol = 2 pmol -> 0.2ul at [10uM]


So, in order:
Count the cells, get 1.250e6 in a 15ml falcon tube and centrifuge. Resuspend in 100uL of nucleofector solution. 
Then, prepare 2 1.5ml Eppendorf tubes with the diluted Luc and DYRK1A (for Luc, take 1.6uM and add 2.4 of Distilled Water - for DYRK1A, 2 and 2). 
Get 5 tubes (names mock, Luc, D10..) and add 20uL of the resuspension. Then add the right volumes of each siRNA + 0.8ul of vector GFP. Mix and add to the cuvettes (to remember which is what, place the tubes in the same order). 
Put in the machine and use the program CM-162.
Get them out, turn it off, wait 10min at room temperature.
Prepare the laminin plate at this point: add 2ml of CM to each well.
Then add 100ul of CM to each cuvette, and take it all with pasteur pipette and drop randomly in the plate.
Change medium after 24hrs. 

MAYBE KEEP TRACK OF CELL EFFICIENCY? 

\subsection{Cell viability assay}
Cell viability was assessed with AlamarBlue to estimate the toxicity of each drug (or IC$_{50}$). The main reagent is Resazurin, a blue and weakly fluorescent dye which emits bright red fluorescence upon its irreversible oxidation. This colorimetric assay is non-toxic which allows repetition of the assay on a three-day period. It is compatible with absorbance and fluorescence reading, which allows us to double-check the results. 

Life Technologies, US

\subsection{Cell staining}
Remove the medium of cells, fix with 4\% PFA for 10 minutes (on shaker), wash 3x5 minutes with PBST (=PBS + 0.1\% Tween). 
Incubate the cells for 1h with blocking solution (at room temperature on the shaker as well).
In a back chamber, add paper + distilled water, flatten, parafin and add 50ul of primary Ab.

Primary AB:

1) (L4) Tuj1 - MOUSE (Biolegend, 801202) - 1:100

2) (G11) Sox2 - RABBIT (CST, 3579P) - 1:400

3) (H2) GFAP - RAT (Invitrogen, 13-0300) - 1:100

Leave overnight and wash 3*5min with PBST. 

Secondary antibodies must simply recognise the animal used in the other antibody. Here:

1) (A1) MOUSE - Alexa 488 A11029 (LifeTechnologies) - 1:1000

2) (C6) RABBIT - Alexa 594 A11037  (LifeTechnologies) - 1:1000

3) (E9) GOAT - Alexa 647 A21247  (LifeTechnologies) - 1:1000

Same as before, put 50ul of Ab on the parafilm, then drop the cover slip cells first on the drop, leave on for 40 min @RT in the black chamber, then rinse 2x5min with PBST and 1x5min with PBS. Put the coverslips on the slides and seal with nail varnish.  


10.05.18 - PROTOCOL FROM LUANA

1. Wash 2x with PBS and fix in 4\% PFA for 15 min @RT, away from light

2. Wash the coverslips 2x with PBS

3. permeabilise with 0.2\% Triton-X in PBS for 10min at RT.

4. was with PBS and block with 3\% BSA in PBS (=blocking solution).

5 - incubate in primary Ab overnight in blocking solution.

6 - wash 3x10min with PBS 

7 - add secondary Ab in block solution (1:1000) and incubate for 1hr @RT

8 - wash 3x10min with PBS (away from light)

9 - mount on slide with DAPI and seal with nail varnish.


\subsection{Western blot}
Preparing samples: http://www.abcam.com/protocols/sample-preparation-for-western-blot
ATTENTION: when you lysate the cells, you want to make sure that you remove ALL the PBS, let the cells dry a little before adding the lysates (in order to get high concentrations of protein since you can only load a restricted volume!).

As written (but RIPA used as base and you add cockaail of inhibitors, blabla), then you let rest for 30min while vortexing every 10 min (for 30s/sample) and finish with 5 pulses of sonication (at power 20 and pulses = 50 - 5 pulses) to get rid of any residue (be sure that the lysate is homogenous). then, centrifuge at 4C at 14000rpm for 10 min. Transfer the supernatant in clean chilled tubes. 

Protein concentration was determined with a BCA assay (standards with Albumin at 2mg/ml, done following the kit ThermoIns)


Must get the sample buffer = lysis buffer (blue) + DTT (stock 20X) which must all be at 4X. 
Complete the lysates according to the Excel sheet (must find the smallest volume and use that for the rest).

cook at 95C for 10 minutes, centrifuge 1min at 7 400 to pull the evaporation, then store at -20C. 

\subsection{Bioreactor simulation}
Using Autodesk Inventor Professional 2019 and Autodesk CFD 2018 with a student licence - maybe COMSOL Multiphysics??


\subsection{Casting bioreactor}
The bioreactor is made of polydimethylsiloxane, or PDMS, which is a form of silicone. It is achieved by mixing the polymers (Sylgrad 184 Silicone Elastomer, Dow Corning) and the cross-linking reagent (Sylgard 184 Silicone Elastomer Curing Agent, Dow Corning) with a 10:1 ratio, which is then poured in the mould. It is then put into a vacuum chamber for one hour to allow encased bubbles to surface up and be removed. Finally, the bioreactor must bake in a 60°C oven for 24 hours.  

\subsection{Image aquisition}
Which microscope(s) was(ere) used.
 (Nikon TI-710 and TI-2000).


\subsection{Radiation}
- cahnged the medium of all cells before radiation every time. (IF ADDING DRUGS, should add right before the radiation rather than after, that means no need to change the milieu after)

- briefly explain how x-ray generators work, to factor in the heel effect (check dans les favoris, le site est super clair nde-ed). 
- explain why we chose x-rays and not alpha-particles for instance: x-rays are the most relevant form of radiation in the in vitro context since the purpose would be to transfer such findings in the clinic, where x-rays are still the standard treatment.
- use of 225kv and 17mA on the EXP 1 (96 well plate, for 1.25min - more or less 2Gy).
- use of 250kV and 12mA on the EXP 2 (24 well plate, exactly 2Gy).

"PROTOCOL":
- new 'machine' to automate the movement of the plate depending on how it must be radiated (so that one needn't enter the room every time to change the placement of the lead - ultimately renders the experiments a lot more reproducible). Still in development so we chose to sacrifice the 8th column in half. 


\subsubsection{Dosimetry}
Initial step was to put the motors and set them to the plate+field size. Then add a film (just to get a feel of the dose distribution) --> got a good distribution (cannot see a heel effect just by looking at the film transcript).

Heel effect: The end result is that the field intensity towards the cathode is more than that towards the anode (due to electrons traversing the target and having to escape it - many are resorbed). Described in my book. 
This is compensated (partially) by a 'heel effect compensation' filter (which also hardens the beam to get rid of 'useless' soft x-rays) which is made of concentric copper rings. 
 

Use of collimator to get a fairly high dose-rate while limiting scatter and achieving a homogeneous dose distribution.

\subsubsection{Protocol?}  


\section{Results}
\subsection{Bioreactor development}
Various simulations:


\subsection{DYRK1A knockdown in traditional cell culture}
\includegraphics[width=\textwidth]{Figure 1.pdf}




- AlamarBlue, SiRNA green images



-  

\subsection{Engineering and biology}


\section{Discussion}
\subsection{Bioreactor development}
\subsection{DYRK1A inhibitors in traditional cell culture}
\subsection{Engineering and biology}
\subsection{Conclusion and outlook}

\section{Conclusion}





\section{Illustrations}
- in Ionescu review (2012), FIG.1 -- the interaction of DYRK1A and its inhibitor.
- draw the Heel effect? 


%\par\noindent\rule{\textwidth}{0.4pt}



%
%\bibliographystyle{unsrt_custom.bst}
%\bibliography{Collection.bib}

\end{document}

